README.md

```diff
@@ -1,3 +1,17 @@
+# End of Life Notice
+We are announcing our decision to stop development of the MongoDB server-side Swift driver. No further development, bug fixes, enhancements, documentation changes or maintenance will be provided by this project and pull requests will no longer be accepted.
+
+There are still ways to use MongoDB with Swift:
+
+- Use the MongoDB driver with server-side Swift applications as is
+- Use the [MongoDB C Driver](https://www.mongodb.com/docs/drivers/c/) directly in your server-side Swift projects
+- Usage of another community Swift driver, mongokitten
+
+Community members and developers are welcome to fork our existing driver and add features as you see fit - the Swift driver is under the Apache 2.0 license and source code is available on GitHub. For those developing client/mobile applications, MongoDB offers the [Realm Swift SDK](https://www.mongodb.com/docs/realm/sdk/swift/) with real time sync to MongoDB Atlas.
+
+We would like to take this opportunity to express our heartfelt appreciation for the enthusiastic support that the Swift community has shown for MongoDB. Your loyalty and feedback have been invaluable to us throughout our journey, and we hope to resume development on the server-side Swift driver in the future.
```

Comment:
> we hope to resume development on the server-side Swift driver in the future.
This seems in contrast to EOLing the driver. Should we title this section something like "Development Pause" instead?